ironfish-cli/src/utils/language.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { LanguageCode } from '@ironfish/rust-nodejs'
+import inquirer from 'inquirer'
+
+// export of napi enum is a const enum which doesn't support anything but value access
+// https://github.com/napi-rs/napi-rs/issues/1345
+export enum Language {
+  English = LanguageCode.English,
+  ChineseSimplified = LanguageCode.ChineseSimplified,
+  ChineseTraditional = LanguageCode.ChineseTraditional,
+  French = LanguageCode.French,
+  Italian = LanguageCode.Italian,
+  Japanese = LanguageCode.Japanese,
+  Korean = LanguageCode.Korean,
+  Spanish = LanguageCode.Spanish,
+}
+
```

Comment:
Any cleaner way to represent these types? Three separate values seems unfortunate to me

Reply:
Thank you @danield9tqh for the great suggestions :)